Assets/Plugins/FMOD/Resources/FMODStudioSettings.asset

```diff
@@ -46,7 +46,7 @@ MonoBehaviour:
     Value: 0
   OverlaySettings:
   - Platform: 1
-    Value: 1
```

Comment:
is this file supposed to be committed ?

Reply:
Yes Nick said he intentionally changed that in voice chat

Reply:
Yes that change was intentional. I disabled the fmod debug window.